src/plugin.js

```diff
@@ -256,6 +256,27 @@ const onPlayerReady = (player, emeError) => {
   setupSessions(player);
 
   if (window.MediaKeys) {
+    player.on('ended', () =>{
+
+      if (videojs.browser.IS_FIREFOX) {
```

Comment:
We only get `encrypted` events on replay and seek-back after content ends on Chrome. Since this doesn't happen on Firefox, https://github.com/videojs/videojs-contrib-eme/blob/main/src/plugin.js#L262-L264 is never called. So a fake encrypted event is necessary here.

Reply:
I think a test for this behavior would be good to have.

Reply:
Unit testing this behaviour is challenging as `handleEncryptedEvent` and all of it's side effects are exported as immutable `const`. To be able to proxy (and therefore spy) it, in [e99939f](https://github.com/videojs/videojs-contrib-eme/pull/225/commits/e99939f64ba119507fd16e1bf9e62fca797277f7) I've changed the definition from `const` to `function`. 

Ideally I think this behaviour would be best covered by an integration test as it's browser-specific and what we actually want to test is whether playback is still possible after an `ended` event , regardless of browser. The actual implementation required to achieve that shouldn't matter  - but we're not set up for integration testing in this repo and to set something like that up would be well beyond the scope of this PR.

FWIW, I have tested playback in Firefox with and without `handleEncryptedEvent` being explicitly called and this  code block does seem to be working as intended.

@adrums86 does the test I've added cover what you you had in mind?

Reply:
Yeah this is all I was looking for. Thank you @Essk! Agree completely about this being an integration test ideally/eventually.